src/WebJobs.Extensions.DurableTask/Listener/TaskActivityShim.cs

```diff
@@ -122,6 +127,11 @@ public override string Run(TaskContext context, string input)
             throw new NotImplementedException();
         }
 
+        internal void SetTaskEventId(int taskEventId)
```

Comment:
When I first encountered `taskEventId`, I thought it referred to ETW / logger Event IDs N ow it's clear that it serves as an ID to differentiate between activity executions.

A short IntelliSense-friendly comment on the `taskEventId` instance variable could help me, and future newcomers, avoid this confusion :)